### ISSUE TITLE
Preparing to become a charity in the UK

### ISSUE THREAD

**Issue by drjwbaker** [programminghistorian/jekyll#1671]
Back when we started looking into becoming a legal entity (May 2019, after dropping the 'budget line' idea I'd been working on for a year) https://github.com/programminghistorian/jekyll/issues/1301 one of the options initially proposed was becoming a charity.

We didn't do this at the time because we didn't meet to key criteria: 1) we didn't have a firmly agreed charitable purpose, and 2) we were unsure if we'd meet the £5k per year income needed to be a charity in the UK.

Nevertheless our solicitors recommended a structure which could easily be rolled into a charity and to keep reviewing if we need to become a charity, should that time come https://github.com/programminghistorian/jekyll/issues/1301#issuecomment-513861381

Now is the time to start preparing for that time. We have a projected Patreon income of ~£800 per year if nothing changes (it will). We have one IPP member https://github.com/programminghistorian/jekyll/issues/1625 and plans https://github.com/programminghistorian/jekyll/issues/1658 to get more. And we have a business plan https://github.com/programminghistorian/jekyll/issues/1657 predicated on getting more money so that we can support the publications. All this means if we get 3 more sign-ups, we're very close to the threshold of **having to** become a charity.

On **having to**:

> [Usually, you must register with the Charity Commission if your charity is based in England or Wales and has over £5,000 income per year. The commission will take action to secure compliance if it identifies a charity which isn’t registered but should be.](https://www.gov.uk/guidance/how-to-register-your-charity-cc21b)

Despite the potential of being forced into it, I see becoming a charity as A Good Thing (for one, we can't accidentally go all commercial and piss off all our nice charitable donors like [KU](https://blogs.openbookpublishers.com/open-book-publishers-statement-on-knowledge-unlatched-and-the-open-research-library/) or [F1000](https://eve.gd/2020/01/12/the-tf-buyout-of-f1000-neutralises-the-plan-s-threat/) have).

The initial actions are:

- [x] 1. **@drjwbaker** To figure out how officious the Charity Commission are on enforcing compliance. 
- [x] ~~2. Agree our purposes now so we don't have to rush them later. See https://www.gov.uk/guidance/how-to-write-charitable-purposes. This can be more or less the same as the [ProgHist Ltd objects (p21 of the bottom pdf)](https://beta.companieshouse.gov.uk/company/12192946/filing-history).~~ Update 10/06/20: replaced with timeline below https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-642064734
- [x] ~~3. Begin identifying potential trustees. See https://www.gov.uk/guidance/trustee-board-people-and-skills?step-by-step-nav=3dd66b86-ce29-4f31-bfa2-a5a18b877f11. *We should do this via email rather than on here*~~ Update 05/06/2020: @acrymble and I agree the we don't need to do this, as the Directors will be the Trustees be default (see, for example, listing of the Black Cultural Archives at [Companies House](https://beta.companieshouse.gov.uk/company/03113148/filing-history) and at the [Charity Commission](https://beta.charitycommission.gov.uk/charity-details/?regid=1051087))
- [x] ~~4. Begin drafting a governing document. See the model document at https://www.gov.uk/guidance/how-to-write-your-charitys-governing-document?step-by-step-nav=3dd66b86-ce29-4f31-bfa2-a5a18b877f11#content~~ Update 10/06/20: replaced with timeline below https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-642064734

I can do some of these but - frankly - not all of them, what with managing what we have in the meantime. Does anyone have the capacity to help? I suspect we won't have the act before we hit 12 months of the company (October), but I think it is sensible to prepare.

@acrymble: apologies for not thinking of this in terms of your business plan, it shouldn't fundamentally change it, I hope.

**Comment by drjwbaker** [comment]
I should add, the company remains in place even when we transition to a charity. So none of that work is lost. We are just doing one at a time rather than both simultaneously!

**Comment by drjwbaker** [comment]
Update on 1: I've asked Keith Lawrey at the Foundation of Science and Technology (who generously helped us incorporate) for advice: if anyone will know, Keith will.

**Comment by acrymble** [comment]
My understanding is that we have the option but not an obligation to become a charity

**Comment by drjwbaker** [comment]
> The commission will take action to secure compliance if it identifies a charity which isn’t registered but should be.

When we first incorporated, Keith informed me that they can enforce this on non-profit companies that 'act like' charities. Just now, I've asked his opinion on the likelihood of that. If IPP ramps up, we may need a legal opinion on whether we are sufficiently 'acting like' a charity.

**Comment by drjwbaker** [comment]
Update: I've been advised that the Charity Commission are unlikely to jump on us the moment we cross the financial threshold. Nevertheless, I will book in some time to start some modest planning as being a charity has some knock-on effects that might change our planning (notably, being unable to move 'profits' to members for services rendered without getting the right legal justifications in place).

**Comment by acrymble** [comment]
That needs a conversation then, because I am motivated in part by wanting to make sure people are paid for their work. Especially people with humanities backgrounds who tend to believe they should give work away for free

**Comment by drjwbaker** [comment]
Agreed. I'll take a look later this week at what changes if we become a charity and report back.

**Comment by drjwbaker** [comment]
(of course, charities can pay people for their work, I think it is just a question of who and how that might conflict with our current set up)

**Comment by acrymble** [comment]
I believe there are training courses you can take for learning how to manage charities. Maybe we need to think about that. Or some of the team anyway.

**Comment by acrymble** [comment]
@drjwbaker is there anything here you need help with in order to action this ticket? Or is the scope of the ticket actionable?

**Comment by drjwbaker** [comment]
@acrymble I think we established since I originally posted this, that we have time. On the other actions listed above my thoughts are:

> 2. Agree our purposes now so we don't have to rush them later. See https://www.gov.uk/guidance/how-to-write-charitable-purposes. This can be more or less the same as the ProgHist Ltd objects (p21 of the bottom pdf).

This is starting to clarify as the Ltd runs (I note your item on team structure in https://github.com/programminghistorian/jekyll/issues/1729). Perhaps we can take some time to consolidate this in the early-Autumn?

> 3. Begin identifying potential trustees. See https://www.gov.uk/guidance/trustee-board-people-and-skills?step-by-step-nav=3dd66b86-ce29-4f31-bfa2-a5a18b877f11. We should do this via email rather than on here

Perhaps Ltd could start this at our next Directors meeting? https://github.com/programminghistorian/jekyll/issues/1748 (starting with the sort of people we want, rather than who)

> 4. Begin drafting a governing document. See the model document at https://www.gov.uk/guidance/how-to-write-your-charitys-governing-document?step-by-step-nav=3dd66b86-ce29-4f31-bfa2-a5a18b877f11#content

I think this flows from 2., and we have most of what we need in place from incorporation. Taking a first stab at this is a task that someone will need to put their hand up to do a first draft of, perhaps this summer.

**Comment by acrymble** [comment]
@drjwbaker can you add this to the Ltd meeting agenda then please?

I think we may need to break this into a 'milestone' of several smaller tasks. Otherwise I think this ticket will remain open for a year or more.

**Comment by drjwbaker** [comment]
Agreed. The milestones I envisage are at https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-633875139

**Comment by drjwbaker** [comment]
Update: @acrymble and I discussed this today whilst working through https://www.resourcecentre.org.uk/information/charity-reporting-and-accounts/ I have taken away an action to break down the concrete steps we need to take, so that we can construct a timeline. I will block out some time for this next week.

**Comment by drjwbaker** [comment]
Our aim to register as a charity in the UK with the Charity Commission once our annual income exceeds £5,000 (which will happen this summer). @acrymble and I have established that we can keep our Company Limited by Guarantee to do this, and then just register with the Charity Commission separately, meaning that existing Ltd Directors will also be Trustees of the Charity (see, for example, Black Cultural Archives at [Companies House](https://beta.companieshouse.gov.uk/company/03113148) and at the [Charity Commission](https://beta.charitycommission.gov.uk/charity-details/?regid=1051087)). 

## Actions and Timeline

- [x] Agree our Charitable Purposes. This can be more or less the same as the ProgHist Ltd objects ([p21 of the Articles of Association](https://github.com/programminghistorian/project-and-business-archive/tree/master/business/Incorporation/memo-of-association)) but we have to get them right as they are hard to change later. The purposes have to be acceptable to the Charity Commission, so there is guidance on writing them at [gov.uk](https://www.gov.uk/guidance/how-to-write-charitable-purposes). **ACTIONS**:
	- [x] Write draft by *late-June*./1671#issuecomment-666347374)
	- [x] Agree by *late-August*.
- [x] ~~Write a [Governing Document (CC22b)](https://www.gov.uk/guidance/how-to-write-your-charitys-governing-document) (note: this includes Charitable Purposes). Note, the [Articles](https://www.gov.uk/government/publications/setting-up-a-charity-model-governing-documents) can be heavily based on our existing [Articles of Association](https://github.com/programminghistorian/project-and-business-archive/tree/master/business/Incorporation/memo-of-association) **ACTIONS**:~~ Not required, see [comment](https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-666347374)
	- [x] ~~Write draft [Articles](https://www.gov.uk/government/publications/setting-up-a-charity-model-governing-documents) by *late-July*.~~ Not required, see [comment](https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-666347374)
	- [x] ~~Write [Memorandum](https://www.gov.uk/government/publications/setting-up-a-charity-model-governing-documents) by *late-July*.~~ Not required, see [comment](https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-666347374)
- [x] ~~Are we going to be a grant making body? If so, we need criteria ([see p6 here](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/557944/Apply_to_register_a_charity_questions.pdf).~~ Decision not to be in the first instance.
- [ ] Trustees need to [complete declaration forms](https://www.gov.uk/government/publications/confirmation-of-charity-trustee-eligibility). **ACTION**:
	- [ ] complete by end of year 1 of the Ltd (6 Sept 2020).
- [ ] Register with the Charity Commission: [questions](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/557944/Apply_to_register_a_charity_questions.pdf) + [video](https://www.youtube.com/watch?v=iYi_hachIgo&rel=0) **ACTION**:
	- [ ] complete by end of year 1 of the Ltd (6 Sept 2020).

## Implications

- [ ] [Annual return](https://www.gov.uk/guidance/prepare-a-charity-annual-return). This becomes more detailed if income over £10k per annum, and more detailed still if income over £25k per annum (see [here](https://www.gov.uk/guidance/prepare-a-charity-annual-return#what-you-need-to-submit))
	- Note: income from outside the UK and spend outside the UK needs to be specified in the return. **This will increase admin**
- [ ] Who we can pay changes. E.g. Trustees, see [points 6.2 and 6.3](https://www.gov.uk/government/publications/trustee-expenses-and-payments-cc11/trustee-expenses-and-payments#employing-a-trustee-or-connected-person)
	- Note: both if and what Trustees were paid is part of the return. **This will increase admin**
- [ ] We will have responsibility to ensure that Trustees know what [Conflicts of Interest](https://www.gov.uk/government/publications/conflicts-of-interest-a-guide-for-charity-trustees-cc29/conflicts-of-interest-a-guide-for-charity-trustees#preventing-conflicts-of-interest-from-affecting-decision-making-legal-requirement) are and that they acknowledge those conflicts when they arise. **This will increase admin**

**Comment by drjwbaker** [comment]
I will action these in the first instance, with support from @acrymble. I've scheduled the work in my diary and will report back as/when.

**Comment by drjwbaker** [comment]
## Draft Charitable Purposes

Per https://github.com/programminghistorian/jekyll/issues/1671#issuecomment-642064734 we need 'Charitable Purposes' to register with the (UK) Charity Commission. So I've put a little time into figuring out what this should be.

These [should say](https://charitysetup.org.uk/checklist/write-your-charitable-objectives/#:~:text=A%20charity%20must%20have%20one,human%20rights%20and%20community%20development.):

- WHAT our charity is going to do
- WHERE we want to do it
- WHO will benefit
- And HOW

Our current 'object' (the non-profit company equivalent of charitable purposes) in our [Articles of Association](https://github.com/programminghistorian/project-and-business-archive/tree/master/business/Incorporation/memo-of-association) are:

> The object for which the Company is established is to advance the education of the public in the humanities in particular in the use of digital tools and techniques and to promote research for the public benefit in all aspects of that subject and to publish the useful results.

This seems to meet the criteria:

- WHAT: 'advance the education [..] in the use of digital tools and techniques and to promote research for the public benefit in all aspects of that subject'
- WHERE: 'of the public'
- WHO: 'in the humanities in particular'
- HOW: 'to publish the useful results'

Looking at [examples](https://www.gov.uk/government/publications/example-charitable-objects/example-objects-advancement-of-education) I see that Girlings - our lawyers during incorporation - have already based our objects on language at gov.uk (see last example), which is good, given that they said they'd written our Articles of Association in such a way that we could easily slide over into being a charity!

Anyway, result of all of this is that I propose only a light rewording for the charitable purposes:

> To advance the education of the public in the humanities in particular in the use of digital tools and techniques and to promote research for the public benefit in all aspects of that subject and to publish the useful results.

Note: the objects of the Ltd remain as the Ltd will underpin the charity (that is how things are structured in the UK)

**Comment by drjwbaker** [comment]
Just documenting here that I'm a little confused.

Looking at the [guidance](https://www.gov.uk/guidance/how-to-write-your-charitys-governing-document), I know what we want to be (a charity company) and what we need (memorandum and articles of association), but I don't know if we need to change our articles of association for the Ltd or write new articles of association for the charity only. If we do, I think we need to notify Companies House https://www.gov.uk/make-changes-to-your-limited-company/constitution-and-articles-of-association

I'm going to ask for a little help from the person who helped us incorporate.

**Comment by drjwbaker** [comment]
Update. [Keith](https://www.foundation.org.uk/About/The-Team) was extraordinarily helpful.

In short, as we want to register as a [Charitable Company (Limited by Guarantee)](https://www.gov.uk/guidance/charity-types-how-to-choose-a-structure?step-by-step-nav=3dd66b86-ce29-4f31-bfa2-a5a18b877f11) we don't need to amend our Articles of Association, so long as the purposes are charitable, which I believe they are.

Instead, we just register with our documents submitted to Companies House. If the Charity Commission don't like our purposes, we agree new ones with the CC, then change the Articles at Companies House, then upload the docs to CC.

So we can skip some steps and move straight to Declaration Forms and registering!

**Comment by acrymble** [comment]
At @drjwbaker 's request I'm asking our board members to sign the relevant paperwork. Using this to keep track of progress as we all have to sign the same file:

- [x] Riva 
- [x] Sarah 
- [x] Sofia 
- [x] Anna Maria 
- [x] James (currently in possession)
- [x] Adam

**Comment by drjwbaker** [comment]
@acrymble: I've worked through most of this today. Still a couple of bits remaining. Might you have time later in the week to look at the application to check that you are happy it is a fair representation of PH?

**Comment by drjwbaker** [comment]
Done.

Note:

> The registration division Is currently experiencing exceptionally high volumes of work And we aim to be in touch with your nominated contact for the application within 40 working days (8 weeks).

@amsichani @svmelton @acrymble @spapastamkou @rivaquiroga You should receive a copy of the application via email. Thanks for your help putting it together.

**Comment by drjwbaker** [comment]
Application added to private archive https://github.com/programminghistorian/project-and-business-archive/blob/master/charity/application/2020-09-04_5163603_OLAR_Registration_503525_01.pdf

**Comment by drjwbaker** [comment]
Our charity application was rejected as our submission had some problems (as opposed to us being rejected for not being a charity). Full details in the shared gmail. The main pieces of work are:

- [ ] 1. Revise of Articles of Association so that they say the name of the charity and are closer to the [model document](https://www.gov.uk/government/publications/setting-up-a-charity-model-governing-documents).
- [ ] 2. Provide more detailed proof of annual income over £5k.
- [ ] 3. "In addition, the bank statement relates to an account for a company called 'PROGHIST LTD' which the applicant confirms in their application is 'connected' a separate company and not the entity which is applying for charitable status.  We therefore require evidence of income in the name of the organisation applying for charitable status which demonstrates that it meets the minimum income requirement."
	- this strikes me as a bigger issue and not something I know how to resolve, because it implies that we need another bank account and to unpick all the finance work we've put in place thus far. I'm probably going to recommend we seek third party advice on this one.
- [ ] 4. We need to provide more detail on the connection between the Ltd and the Charity.
- [ ] 5. The Trustee Declaration must include the name of the Charity.

In short, I think we are charitable and we still have an obligation to register ("[You must apply to register your charity if its income is at least £5,000 per year](https://www.gov.uk/setting-up-charity/register-your-charity)") but that we probably need some expert advice to help us rework what we have, especially on 3 and 4.

My focus this month is on getting the annual accounts sorted so I might have to put this on the back burner for now.

Thanks all for your efforts so far.

**Comment by drjwbaker** [comment]
Note, I'm picking this up with our accountants West & Berry, and will resume work on this in February (at which point I'll reopen the ticket)

**Comment by drjwbaker** [comment]
A revised application has now been submitted. We can expect an answer in 2/3 months. If any small issues arise, West & Berry will handle/triage them.

**Comment by drjwbaker** [comment]
Note that application came back positive, but with some queries. These were resolved by me, with support of Ltd Directors and West & Berry.
